vgen/src/main/java/io/gapi/vgen/ruby/RDocCommentFixer.java

```diff
@@ -28,15 +31,40 @@
   private static final Pattern CLOUD_LINK_PATTERN = Pattern.compile(
       "\\[([^\\]]+)\\]\\(([^\\)]+)\\)");
   private static final Pattern PROTO_LINK_PATTERN = Pattern.compile(
-      "\\[([^\\]]+)\\]\\[[^\\]]+\\]");
+      "\\[([^\\]]+)\\]\\[[^\\]]*\\]");
```

Comment:
Probably better to be done in another CL -- filed as https://github.com/googleapis/toolkit/issues/38